src/projects/apps/designer/size.md

```diff
@@ -0,0 +1,14 @@
+# Size
+
+This property is useful to control the size of the component. The **height** and **width** properties are used to set the height and width of a component. It sets the height/width of the area inside the padding, border, and margin of the component.
```

Comment:
It does not set the height _within_ the padding - everything within apps inherits `box-sizing: border-box` (unless they explicitly opt-out), meaning that specified heights include the padding.